tests/run-macros/term-show/Test_2.scala

```diff
@@ -25,5 +37,17 @@ object Test {
       |  }
       |  ()
       |}""".stripMargin)
+
+    assert(showTree("A")
```

Comment:
It would be better to move this check into a checkfiles. They are simpler to update/debug if the format changes.

```suggestion
    println(showTree("A"))
```

Add `tests/run-macros/term-show.check` then run `scala3-bootstrapped/testCompilatation tests/run-macros/term-show`. Follow the instructions of the error message; the `mv` command printed will update the checkfile.

Reply:
That's what we had before the merge with main, but https://github.com/lampepfl/dotty/pull/16968 used a different style and a followed what was already in main.

@nicolasstucki Should I refactor both asserts to use checkfiles for consistency ?

Reply:
Done in https://github.com/lampepfl/dotty/pull/17952/commits/6afd86d6ee859f180581c304998ce4d0e337d423.